content/Coalesce.Starter.Vue.Web/package.json

```diff
@@ -9,27 +9,28 @@
     "lint": "vue-cli-service lint"
   },
   "dependencies": {
-    "coalesce-vue": "^2.1.0-alpha-310",
+    "coalesce-vue": "^2.2.0-alpha-509",
     "vue": "^2.5.16",
-    "vue-class-component": "^6.0.0",
-    "vue-property-decorator": "^6.0.0",
-    "vue-router": "^3.0.1"
+    "vue-class-component": "^6.2.0",
+    "vue-property-decorator": "^7.0.0",
+    "vue-router": "^3.0.1",
+    "vuetify": "~1.0.16"
   },
   "devDependencies": {
```

Comment:
Latest @vue/cli-* is 3.0.3 now, and coalesce-vue requires typescript 3.0+

Reply:
Changed those versions, but when I run `yarn build` (after running `dotnet coalesce`) I get this error:
```
ERROR in /home/thor/Dev/intellitect/Coalesce.Vue.Template/content/Coalesce.Starter.Vue.Web/src/views/CoalesceExample.vue
26:26 Argument of type '(this: any, c: ApplicationUserApiClient) => (this: HelloWorld) => Promise<AxiosResponse<ListResult<ApplicationUser>>>' is not assignable to parameter of type 'TInvoker<[], Promise<AxiosResponse<ListResult<{}>>>, ApplicationUserApiClient>'.
  Type '(this: HelloWorld) => Promise<AxiosResponse<ListResult<ApplicationUser>>>' is not assignable to type 'Promise<AxiosResponse<ListResult<{}>>>'.
    Property 'then' is missing in type '(this: HelloWorld) => Promise<AxiosResponse<ListResult<ApplicationUser>>>'.
    24 | 
    25 |   private loadApplicationUsers = new ApplicationUserApiClient()
  > 26 |     .$makeCaller("list", c => function(this: HelloWorld) { return c.list() })
       |                          ^
    27 |     .setConcurrency("cancel");
```
...and I'm not competent in Vue enough to know what's wrong :)

Reply:
Change that line to `.$makeCaller("list", function(this: HelloWorld, c) { return c.list() })`

This was a change in `coalesce-vue` using new TS 3.0 features, which is why we require TS 3.0.

Reply:
Cool, should be 5 by 5 now.